webknossos/webknossos/dataset/_downsampling_utils.py

```diff
@@ -314,7 +314,7 @@ def downsample_cube_job(
         tiles = product(
             *(
                 list(range(0, math.ceil(len / buffer_edge_len)))
-                for len, buffer_edge_len in zip(shape[-3:], buffer_shape)
+                for len, buffer_edge_len in zip(shape[-3:], buffer_shape) # noqa:  A001 Variable `len` is shadowing a Python builtin
```

Comment:
fix directly by renaming to length? should only be used in 316 and 317.

Reply:
Done